zuul.d/pipelines.yaml

```diff
@@ -52,4 +52,7 @@
         # Verified: 1
     failure:
       gerrit: {}
-        # Verified: -1
\ No newline at end of file
+        # Verified: -1
+- pipeline:
```

Comment:
is this intentional or did it slip in this review by accident?

Reply:
Good catch, it was leftovers from discussing what it would like to have series specific recheck queues